app/controllers/users_controller.rb

```diff
@@ -1,9 +1,11 @@
 class UsersController < ApplicationController
   def index
     @users = User.all
+    render json: @users
   end
 
   def show
     @user = User.find(params[:id])
+    render json: @user
   end
 end
```

Comment:
- [ ] Please create your API endpoints in a different directory, You should use `/api` or `api/v1`, You will also need to create separate routes for those API endpoints. `( It's often a common practice to add /api to URL or api/v1 with the version number in order to identify it as an API-based URL)`.